sink/sink

```diff
@@ -100,20 +100,30 @@ def expand(val, env):
 # Compare two JSON style values to see if one is contained in the other
 # For lists and dict make sure all items in expect are also present in result
 # For any other values a strict comparison is used
-def contains(expect, result):
+def contains(expect, result, debug=True):
```

Comment:
The commit message of the 4th commit is identical to the third one. This should be squashed into the 3rd commit.